### PR TITLE
feat: add docs for keras image encoder

### PR DESCRIPTION
**Pull request by rutujasurve94**
Doc for `kerasimage` encoder

### CODE REVIEW

File: encoders/image/ImageKerasEncoder/README.md [hunk 1]
```diff
@@ -1,3 +1,57 @@
 # ImageKerasEncoder
 
-`ImageKerasEncoder` encodes data from a ndarray, potentially B x (Channel x Height x Width) into a ndarray of `B x D`. Internally, :class:`ImageKerasEncoder` wraps the models from `tensorflow.keras.applications`. https://keras.io/applications/
\ No newline at end of file
+`ImageKerasEncoder` encodes data from a ndarray, potentially B x (Channel x Height x Width) into a ndarray of `B x D`. Internally, :class:`ImageKerasEncoder` wraps the models from `tensorflow.keras.applications`. https://keras.io/applications/
```
Comment (JoanFM) [inline encoders/image/ImageKerasEncoder/README.md:3]: Can u say what the `B` stands for

File: encoders/image/ImageKerasEncoder/README.md [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+
+
+## Usage:
+
+Initialise this Executor specifying parameters i.e.:
+
+| `param_name`  | `param_remarks` |
+| ------------- | ------------- |
+| `model_path`  | the directory path of the model in the `SavedModel` format  |
+| `model_name`  | includes `MobileNetV2`  |
```
Comment (JoanFM) [inline encoders/image/ImageKerasEncoder/README.md:13]: what other possibilities?

File: encoders/image/ImageKerasEncoder/README.md [hunk 3]
```diff
@@ -1,3 +1,57 @@
...
+| `channel_axis`| axis id of the channel, etc.  |
```
Comment (JoanFM) [inline encoders/image/ImageKerasEncoder/README.md:14]: axis of the `color` channel

File: encoders/image/ImageKerasEncoder/README.md [hunk 4]
```diff
@@ -1,3 +1,57 @@
...
+| `pool_strategy` | strategy like `avg` |
```
Comment (JoanFM) [inline encoders/image/ImageKerasEncoder/README.md:15]: strategy for the pooling operation

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: encoders/image/ImageKerasEncoder/README.md (hunk 2 above)
Comment (rutujasurve94) [reply encoders/image/ImageKerasEncoder/README.md]: added other possibilities

**Review by JoanFM — APPROVED** [review]
